api/lib/notifications/notifications-types.js

```diff
@@ -79,8 +79,34 @@ const getDefaultSubscriptions = async (addresses, dataSources) => {
   return subscriptions
 }
 
+const getDefaultEMailSubscriptions = async (addresses, dataSources) => {
+  let subscriptions = []
+
+  for (const { address, networkId } of addresses) {
+    const delegations = await dataSources[
+      networkId
+    ].api.getDelegationsForDelegatorAddress(address)
+
+    delegations.forEach((delegation) => {
+      subscriptions.push(
+        `${delegation.validatorAddress}_${eventTypes.VALIDATOR_COMMISSION}_${networkId}`,
+        `${delegation.validatorAddress}_${eventTypes.VALIDATOR_STATUS}_${networkId}`,
+        `${delegation.validatorAddress}_${eventTypes.SLASH}_${networkId}`,
+        `${delegation.validatorAddress}_${eventTypes.LIVENESS}_${networkId}`
```

Comment:
So this subscribes you by default to your own validators' updates right?

Reply:
Nice

Reply:
When does the Liveness notification get triggered? How many blocks does the validator need to miss?

Reply:
It doesn't get triggered yet. See #4504